packages/babel-preset/CHANGELOG.md

```diff
@@ -5,11 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+
+### Changed
+
+- Update `babel-*` to `7.9.6` [[#155](https://github.com/Shopify/web-foundation/pull/155)]
```

Comment:
Do we want to call out which ones as this doesn't touch all of them

Reply:
Done